src/components/JoinCommunity/index.jsx

```diff
@@ -22,6 +22,11 @@ export default function Community() {
                 <p>Join the ever-growing uptane community for updates, support, and collaboration.</p>
             </div>
             <div className={styles.community_links}>
+                <div className={styles.links}>
+                    {/* <LinkCard link="#" title="Slack" light="img/community_links/slack.svg" dark="img/community_links/slack.svg" /> */}
+                    <LinkCard link="https://discord.gg/uWyT6gDCqx" title="Discord" light="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAeJJREFUSEvdlU9O20AUxr83WMqycALCDcLCSF0lPQHcoFnG3RD2xQzmAGRVh505QckN0lWlGokewZygYVWQzLzqGduM/xAJUljg5Yzn/cbf971nwis/9Mr18Y4BY321/tekfUUYMGE21e7cltP7ermHNe4bxvwscGdPSd2QSArf8v0pMe8BWH84yAmYkkoR4i5A3XxtAUIUHrsHdVAD8MWPNQNHLzKfcRCeuBP7bAUw0j+7ZJyrx5s/G7PoKGdrorcXxckKwPN/RQB9fnZZ6wABx98CVz8BiIX8YRWA+BUGO1sNQJYKxd9bit8sgbbuscKnInWlRCP/ckLg/QogNy33RmK6me9fs0oHU/0x8Q7jMQinNWNLmUqA58dSoG+/GAautW/7w+dhsDMs3vX8mGtf/iMM3IGs2YCG/h3lbBSJGPnxBQG7WVcAs2ngSp9A+ubOpH+qgEcfbED9FlLpwhAiAnoElMnIIdIvvxVjCEIGa/v65YAV4lTI+4aAhzSIDHkfcAIozWyIiMRQmTtligAkzBxlRpKMlnIu3RAwKZqtZVSsaaubrzvK6dmtb6uWGyxDMLuUmA+VjiW+rZ1cLErulXGGRmFeH9N1W6RBSXHPqDSyCy8FrOBt4+g7/mX+L5n+AUAhwxkQhpi9AAAAAElFTkSuQmCC" dark="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAAAXNSR0IArs4c6QAAAeJJREFUSEvdlU9O20AUxr83WMqycALCDcLCSF0lPQHcoFnG3RD2xQzmAGRVh505QckN0lWlGokewZygYVWQzLzqGduM/xAJUljg5Yzn/cbf971nwis/9Mr18Y4BY321/tekfUUYMGE21e7cltP7ermHNe4bxvwscGdPSd2QSArf8v0pMe8BWH84yAmYkkoR4i5A3XxtAUIUHrsHdVAD8MWPNQNHLzKfcRCeuBP7bAUw0j+7ZJyrx5s/G7PoKGdrorcXxckKwPN/RQB9fnZZ6wABx98CVz8BiIX8YRWA+BUGO1sNQJYKxd9bit8sgbbuscKnInWlRCP/ckLg/QogNy33RmK6me9fs0oHU/0x8Q7jMQinNWNLmUqA58dSoG+/GAautW/7w+dhsDMs3vX8mGtf/iMM3IGs2YCG/h3lbBSJGPnxBQG7WVcAs2ngSp9A+ubOpH+qgEcfbED9FlLpwhAiAnoElMnIIdIvvxVjCEIGa/v65YAV4lTI+4aAhzSIDHkfcAIozWyIiMRQmTtligAkzBxlRpKMlnIu3RAwKZqtZVSsaaubrzvK6dmtb6uWGyxDMLuUmA+VjiW+rZ1cLErulXGGRmFeH9N1W6RBSXHPqDSyCy8FrOBt4+g7/mX+L5n+AUAhwxkQhpi9AAAAAElFTkSuQmCC" />
```

Comment:
here is resource for that : https://discord.com/branding

Reply:
Will do that thanks